### PR TITLE
these are the tweaks i made

### PR DESCRIPTION
**Pull request by joeyjumper94**
it fixed the errors these files caused when they were autorun, it also simplified things too.

### CODE REVIEW

**Comment by TimGoll** [comment]
Please take a look at the coding style. You can see the output here: https://ci.neoxult.de/job/TTT2/job/PR-484/1/

**Comment by joeyjumper94** [comment]
that should be fixed now

**Comment by TimGoll** [comment]
Yes, it would work. But we try to have a consitent style and decided on the lua style

File: lua/autorun/server/ttt2_force_download.lua [hunk 1]
```diff
@@ -1,7 +1,7 @@
 -- This file forces clients to download the icons
 -- If you are distributing those files via FastDL, comment out the line below.
 
-if SERVER then
+if engine.ActiveGamemode() == "terrortown" then
```
Comment (TimGoll) [inline lua/autorun/server/ttt2_force_download.lua]: it should still be server only

Comment (saibotk) [reply lua/autorun/server/ttt2_force_download.lua]: That is ensured, as this file is in the /autorun/server folder

Comment (TimGoll) [reply lua/autorun/server/ttt2_force_download.lua]: my bad

File: lua/autorun/ttt_bind_autorun.lua [hunk 2]
```diff
@@ -1,6 +0,0 @@
--- Clientside bind lib
```
Comment (saibotk) [inline lua/autorun/ttt_bind_autorun.lua]: Why did you remove this file?

File: lua/autorun/client/ttt_bind_autorun.lua [hunk 3]
```diff
@@ -1,3 +1,4 @@
+if engine.ActiveGamemode() ~= "terrortown" then return end--block if TTT is not running
 ------------
```
Comment (saibotk) [inline lua/autorun/client/ttt_bind_autorun.lua]: This file should not be moved and the `ttt_bind_autorun.lua` file should still be kept, as it will add the lua file to the files a client has to download.

Comment (Alf21) [reply lua/autorun/client/ttt_bind_autorun.lua]: Why? I don‘t see any reason, autorun files are downloaded automatically afaik

Comment (saibotk) [reply lua/autorun/client/ttt_bind_autorun.lua]: Yes, but we do not have "logic-heavy" files in the autorun folder as of now, and i think that is much cleaner as it is currently, with the bind code in its own folder.

**Comment by saibotk** [comment]
This was done in 655f9e596d646f3e2941a85c60efc6e9ee3123da
